libs/auth/src/drivers/magento/auth.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+import { Observable, of, throwError } from 'rxjs';
+import { switchMap } from 'rxjs/operators';
+import { Apollo } from 'apollo-angular';
+
+import { DaffAuthServiceInterface } from '../interfaces/auth-service.interface';
+import {
+  checkTokenQuery,
+  MagentoCheckTokenResponse
+} from './queries/public_api';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffMagentoAuthService implements DaffAuthServiceInterface {
+  constructor(private apollo: Apollo) {}
+
+  check(): Observable<void> {
+    return this.apollo.query<MagentoCheckTokenResponse>({query: checkTokenQuery}).pipe(
+      switchMap(resp => {
+        if (resp.data.customer.id) {
```

Comment:
I would expect Magento to have a error code on the response, instead of checking a field. Is this not the case?

Reply:
You are correct, I added error handling to this call/